gusto/timestepping/semi_implicit_quasi_newton.py

```diff
@@ -477,3 +485,24 @@ def apply(self, x_in, x_nl, x_out, label):
 
         x_out.assign(x_in(self.field_name))
         x_out += self.xF
+
+    def zero_forcing_terms(self, equation, x_in, x_out, transported_field_names):
+        """
+        Zero forcing term F(x) for non-wind transport.
+
+        This takes x_in and x_out, where                                      \n
+        x_out = x_in + scale*F(x_nl)                                          \n
+        for some field x_nl and sets x_out = x_in for all non-wind transport terms
+
+        Args:
+            equation (:class:`PrognosticEquationSet`): the prognostic
+                equation set to be solved
+            x_in (:class:`FieldCreator`): the field to be incremented.
+            x_out (:class:`FieldCreator`): the output field to be updated.
+            transported_field_names (str): list of fields names for transported fields
+        """
+        for field_name in transported_field_names:
+            if field_name != 'u':
+                logger.info(f'SIQN: Zeroing Implicit forcing for {field_name}')
```

Comment:
I think the `SIQN` in the logged message now needs to be `Semi-Implicit Quasi Newton`